tests/unit/Application/Metadata/JsonGenerator/ArpGeneratorTest.php

```diff
@@ -36,7 +36,7 @@ class ArpGeneratorTest extends MockeryTestCase
 
     public function setUp()
     {
-        $this->attributeRepository = new AttributeRepository(__DIR__ . '/../../../../../app/Resources/metadata');
+        $this->attributeRepository = new AttributeRepository(__DIR__ . '/../../../../../app/config/attributes.json');
```

Comment:
The arp generator should not have been changed in this PR right? Maybe this is an attempt to fix the unit tests? The real fix here should be to use the old AttributeRepository on the ArpGenerator. 

Lets discuss if this should be fixed here, or if this is just some collateral damage we want to live with at this point.